TeamMemberFiles/Wessley.Monnin.txt

```diff
@@ -0,0 +1,5 @@
+Wessley Monnin
+Wessley
+294
+Java
+I have very limited to no experience with Github
```

Comment:
Great job Wessley!

Please make sure to clean up the repo from your side to make sure not to expose any private information to the public